modules/cloudbuild/templates/cloudbuild-pull-request.yaml

```diff
@@ -29,6 +29,19 @@ steps:
               echo "*************************************************"
               terraform init || exit 1
 
+- id: 'tf version'
```

Comment:
testing in cloudnuage.dev (existing install) and terraform.landing.systems (new deploy)